packages/terra-toggle/src/terra-dev-site/test/toggle/OpenToggle.test.jsx

```diff
@@ -17,7 +17,7 @@ class OpenToggle extends React.Component {
     const { isOpen } = this.state;
     return (
       <div>
-        <button type="button" id="trigger-toggle" onClick={this.handleOnClick}>Open Toggle</button>
+        <button type="button" id="trigger-toggle" onClick={this.handleOnClick} aria-expanded={isOpen}>Open Toggle</button>
```

Comment:
Let's add the aria-controls and id similar to `packages/terra-toggle/src/terra-dev-site/doc/example/AnimatedToggle.jsx`